src/formElementPlugins/SummaryViewWrapper.tsx

```diff
@@ -82,18 +48,14 @@ const SummaryViewWrapper: React.FC<SummaryViewWrapperProps> = (props) => {
   )
 
   return (
-    <Grid columns={2}>
+    <Grid columns={1}>
```

Comment:
We don't need to use the Grid here anymore, the ErrorBoundary can be straight at the root level of this component